reference/promise-types/files.markdown

```diff
@@ -767,12 +766,12 @@ itself; in other words, a single destination directory. So `src/subdir/file` wil
 **Example:**
 
 ```cf3
-     body copy_from mycopy(from,server)
-     {
-     source      => "$(from)";
-     servers     => { "$(server)" };
-     collapse_destination_dir => "true";
-     }
+body copy_from mycopy(from,server)
+{
+source      => "$(from)";
```

Comment:
usually attributes are indented  but i can see they were not here and the fixup just fixed the whole block indention, so ok

Reply:
Yep, no autoformatting yet.